segments/wan_ip.sh

```diff
@@ -39,7 +39,7 @@ run_segment() {
 	fi
 
 	if [ -z "$wan_ip" ]; then
-		if wan_ip=$(curl --max-time 2 -s http://whatismyip.akamai.com/); then
+		if wan_ip=$(curl --max-time 2 -s -L https://whatismyip.akamai.com/); then
```

Comment:
I guess it was not clear what i meant. The reason why you get a 302, is the redirect from http to https. So we could either use `-L` to follow the redirect and keep the initial `http`, or we can just directly use `https` in which case we won't need the `-L` anymore.

```suggestion
		if wan_ip=$(curl --max-time 2 -s https://whatismyip.akamai.com/); then
```

Reply:
Uh, I have got your points. But I think keeping `-L` will be helpful if akamai redirect it later.

Reply:
I don't expect any other redirects, as this is a special case due to switching to `https`-only.
If there will be more changes on akamai upstream, we'll address them correctly, but blindly following redirects will waste resources, obfuscate the usage of the wrong url (like you found out) and even introduces a security risk.